app/lib/actions/pulp3/orchestration/repository/update.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module Repository
+        class Update < Pulp3::Abstract
+          def plan(repository, smart_proxy)
+            sequence do
+              plan_action(Actions::Pulp3::Repository::UpdateRepository, repository, smart_proxy)
+              #plan_action(Actions::Pulp3::Repository::UpdateRemote, repository, smart_proxy)
```

Comment:
the more i think about this, the more i think we should update the remote at sync time.  That gives us a lot of advantages, especially when importing a new manifest (where we'd have to update all remotes of RH repos).  Curious your thoughts?

Reply:
I will defer to your gut feeling. I'm still very new to katello/pulp3. Especially if it simplifies things.

Reply:
I think @sjha4 mentioned this before too, thoughts @sjha4?  You may have to layer this pr ontop of https://github.com/Katello/katello/pull/8054 and add the remote update to the pulp3 orchestration sync task.